code/game/gamemodes/objective.dm

```diff
@@ -35,6 +35,8 @@ var/list/potential_theft_objectives = subtypesof(/datum/theft_objective) \
 		return TARGET_INVALID_NOT_HUMAN
 	if(!possible_target.current.stat == DEAD)
 		return TARGET_INVALID_DEAD
+	if(possible_target.key == null)

```

Comment:
`possible_target` IS the mind. It's either this or `possible_target.current--`further magic?